test/Ocelot.AcceptanceTests/StreamContentTest.cs

```diff
@@ -0,0 +1,149 @@
+﻿using System.Security.Cryptography;
+using Microsoft.AspNetCore.Http;
+using Ocelot.Configuration.File;
+
+namespace Ocelot.AcceptanceTests;
+
+public class StreamContentTest : IDisposable
```

Comment:
```suggestion
public class StreamContentTests : IDisposable
```
and rename the file!

Reply:
Fixed in commit https://github.com/ThreeMammals/Ocelot/pull/1972/commits/dcbbab7119180345afaf001fbf3664f64b2fafd7 ✔️